acceptance/framework/consul/helm_cluster.go

```diff
@@ -470,3 +452,25 @@ func defaultValues() map[string]string {
 	}
 	return values
 }
+
+func CreateOrUpdateK8sSecret(t *testing.T, client kubernetes.Interface, cfg *config.TestConfig, namespace, secretName, secretKey, secret string) {
```

Comment:
```suggestion
func CreateOrCheckK8sSecret(t *testing.T, client kubernetes.Interface, cfg *config.TestConfig, namespace, secretName, secretKey, secret string) {
```
I think its not updating a secret, just checking if it exists?